### DISCUSSION TITLE
部分高码率的视频播放不了

### DISCUSSION THREAD

**Discussion by itworkzhang** [AlistGo/alist#386] [Q&A] [answered]
部分视频在谷歌浏览器黑屏，只有声音。在edge浏览器声音都没有，一直反复重新连接。
![image](https://user-images.githubusercontent.com/42572628/149928015-ca75befb-fd75-40ff-9017-af2430abf596.png)

**Comment by xhofe** [comment, marked as answer]
- #227 
- #244 
- #210 
- https://alist-doc.nn.ci/docs/common-question#%E8%A7%86%E9%A2%91%E6%92%AD%E6%94%BE%E4%B8%8D%E4%BA%86
